SampleApp/ButtonView.swift

```diff
@@ -1,6 +1,126 @@
 import SwiftUI
 import Warp
 
+struct ButtonView: View {
+    @State private var type: Warp.ButtonType = .primary
+
+    @State private var buttonTitle = "Title"
+
+    @State private var buttonHasIcon = false
+
+    @State private var buttonSize: Warp.ButtonSize = .big
+
+    @State private var isButtonEnabled = true
+
+    @State private var isButtonFullWidth = false
+
+    @Environment(\.colorScheme) private var colorScheme
+
+    var body: some View {
+        ScrollView(showsIndicators: false) {
+            createButton()
+                .frame(height: 50)
+                .padding(.vertical)
+
+            GroupBox(
+                content: {
+                    Picker("Pick your box style please", selection: $type.animation(.smooth)) {
+                        ForEach(Warp.ButtonType.allCases, id: \.self) { type in
+                            Text(type.name)
+                        }
+                    }
+                    .pickerStyle(.wheel)
+                    .defaultPadding()
+                }, label: {
+                    Text("Type")
+                }
+            )
+
+            GroupBox(
+                content: {
+                    TextField("Write the desired title", text: $buttonTitle)
```

Comment:
If we replace this with `Warp.TextField` will the `colorScheme` be removed? We should use our own components as much as we can, even in the `DemoApp`

Reply:
Good point, updated 👍